spec/factories/annotations.rb

```diff
@@ -0,0 +1,33 @@
+FactoryBot.define do
+  factory :annotation do
```

Comment:
Now that we have a factory for annotations, I think it would be nice to have some unit tests for the `Annotation` model (possibly in a different PR).